src/main/kotlin/uk/gov/dluhc/printapi/database/repository/PrintDetailsRepository.kt

```diff
@@ -63,7 +63,7 @@ class PrintDetailsRepository(client: DynamoDbEnhancedClient, tableConfig: Dynamo
             .expressionNames(mapOf(Pair("#print_status", "status")))
             .expressionValues(
                 mapOf(
-                    Pair(":status", AttributeValue.builder().s(Status.PENDING_ASSIGNMENT_TO_BATCH.toString()).build())
+                    Pair(":status", AttributeValue.builder().s(status.toString()).build())
                 )
```

Comment:
Minor (and it doesn't matter because this class will be deleted soon, but you can do this neater without a `Pair` like this:
```
                mapOf(
                    ":status" to AttributeValue.builder().s(status.toString()).build()
                )
```